File: py/IEC61968/Work/BreakerMaintenanceKind.py
from enum import Enum


# Converted by an OPENAI API call using model: gpt-3.5-turbo-1106
class BreakerMaintenanceKind(Enum):
    EXTERNAL_OUT_OF_SERVICE = 0
    INTERNAL_OUT_OF_SERVICE = 1
    INTERRUPTER_OVERHAUL = 2

File: java/IEC61968/Work/BreakerMaintenanceKind.java
package IEC61968.Work;


/**
 * Possible types of breaker maintenance work.
 * @author ppbr003
 * @version 1.0
 * @created 25-Dec-2023 8:45:19 PM
 */
public enum BreakerMaintenanceKind {
	/**
	 * External out-of-service inspection and maintenance.
	 */
	externalOutOfService,
	/**
	 * Internal (interrupter) inspection and maintenance (breaker out of service).
	 */
	internalOutOfService,
	/**
	 * Overhaul of breaker interrupter unit. 
	 */
	interrupterOverhaul
}